packages/common/src/core/schemas/internal/ConfigurableEntity.ts

```diff
@@ -2,3 +2,8 @@
  * Generic interface for a Hub Entity
  */
 export type ConfigurableEntity = Record<string, any>;
+
+/**
+ * Generic interface for a layout card
+ */
+export type ConfigurableCard = Record<string, any>;
```

Comment:
Also thinking we should make this type more explicit. For now it would just be: 

```ts
export type ConfigurableCard = {
  themeColors: string[]
}
```

And we can add additional card config properties as we need.

Reply:
Also wondering if this should be in its own `ConfigurableCard.ts` file rather than bundled into this file?